frontend/src/ui/Blocks/Series.tsx

```diff
@@ -162,7 +189,7 @@ const ReadySeriesBlock: React.FC<ReadyProps> = ({
             {...{ basePath, event }}
         />);
 
-    const eventsUI = events.length === 0
+    const eventsUI = !eventsNotEmpty
```

Comment:
Added this check because before we would show the `no-events` text even if there were upcoming events present